kayenta-core/src/test/groovy/com/netflix/kayenta/metrics/CanaryScopeSpec.groovy

```diff
@@ -58,7 +57,9 @@ class CanaryScopeSpec extends Specification {
 
   private ObjectMapper myObjectMapper() {
     ObjectMapper objectMapper = new ObjectMapper();
-    KayentaConfiguration.configureObjectMapperFeatures(objectMapper);
+    KayentaSerializationConfigurationProperties kayentaSerializationConfigurationProperties = new KayentaSerializationConfigurationProperties();
+    kayentaSerializationConfigurationProperties.setWriteDatesAsTimestamps(false);
```

Comment:
It looks like the property defaults to `false`, so does this setting have any effect?

Does the test fail if this property is set to `true`?